templates/repo/diff/options_dropdown.tmpl

```diff
@@ -14,5 +14,20 @@
 		{{end}}
 		<a id="expand-files-btn" class="item">{{.locale.Tr "repo.pulls.expand_files"}}</a>
 		<a id="collapse-files-btn"class="item">{{.locale.Tr "repo.pulls.collapse_files"}}</a>
+		{{if .Issue.Index}}
+			{{if .ShowOutdatedComments}}
+				<a class="item" href="?show-outdated=false">
```

Comment:
What's interesting about these hrefs is that when there are already options in the URL, like `?style=split` clicking these will override the whole query string. The page still seems to remember the split style thought. I wonder if we should actually merge the query string here with JS, should check how the other buttons do it.

Reply:
Yep - just checked the code: if you're logged in it'll use the stored value for split styl. if you're not logged in it'll default.

I'll change the link generation like it's done in whitespace_dropdown https://github.com/go-gitea/gitea/blob/main/templates/repo/diff/whitespace_dropdown.tmpl#L11

Reply:
Yep - just checked the code: if you're logged in it'll use the stored value for split styl. if you're not logged in it'll default.

I'll change the link generation like it's done in whitespace_dropdown https://github.com/go-gitea/gitea/blob/main/templates/repo/diff/whitespace_dropdown.tmpl#L11

Reply:
I see, so there's at least 3 URL options now that need to be pictured in all these links. Should eventually move it to JS but I guess I can accept a 3rd being added now.

Reply:
Fixed